contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -44,20 +50,10 @@ class ExpectColumnValuesToBePresentInAnotherTable(QueryExpectation):
      }
     ExpectColumnValuesToBePresentInAnotherTable will PASS for example 1 and FAIL for example 2.
 
-    Also, the `template_dict` parameter we would use for the Expectation on `order_table_1` and `order_table_2` would
-    look like the following:
-
-    "template_dict" = {
-        "foreign_key_column": "CUSTOMER_ID",
-        "foreign_table": "customer_table",
-        "primary_key_column_in_foreign_table": "CUSTOMER_ID",
-    }
-
-     Args:
-         template_dict: dict containing the following keys:
-             foreign_key_column: foreign key-column of current table that we want to validate.
-             foreign_table: foreign table.
-             primary_key_column_in_foreign_table: key column for primary key in foreign table.
+    Args:
+        foreign_key_column: foreign key-column of current table that we want to validate.
```

Comment:
Nit: we prob don't need to hyphenate `key-column` here since it's not hyphenated on line 56